lib/converters/converter-v2-to-v1.js

```diff
@@ -116,6 +116,11 @@ _.extend(Builders.prototype, {
         return _.map(item.request.body[mode], function (elem) {
             elem = _.clone(elem);
 
+            if (elem && elem.type === 'file' && _.has(elem, 'src')) {
+                elem.value = elem.src;
```

Comment:
@czardoz In that case, the V1 schema will also have to be fixed to specify `src` for files instead of `value`.